.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+
+New Feature
+
+- [ ] Code conforms to style guide
+- [ ] Doctests
+- [ ] Unit-tests
+- [ ] Documentation
+- [ ] New line item in [releases.md](releases.md)
+- [ ] Info added to `literature` folder
```

Comment:
Let's not encourage the stupid microsoft contraption of `folder`. It's a `directory`.

Reply:
😄